becquerel/tools/nndc.py

```diff
@@ -226,65 +229,63 @@ def _parse_float_uncertainty(x, dx):
     """
 
     if not isstring(x):
-        raise NNDCRequestError('Value must be a string: {}'.format(x))
+        raise NNDCRequestError("Value must be a string: {}".format(x))
     if not isstring(dx):
-        raise NNDCRequestError('Uncertainty must be a string: {}'.format(dx))
+        raise NNDCRequestError("Uncertainty must be a string: {}".format(dx))
     # ignore percents
-    if '%' in x:
-        x = x.replace('%', '')
+    if "%" in x:
+        x = x.replace("%", "")
     # ignore unknown ground state levels (X, Y, Z, W)
-    for sym in ['X', 'Y', 'Z', 'W']:
-        if '+' + sym in x:
-            x = x.replace('+' + sym, '')
+    for sym in ["X", "Y", "Z", "W"]:
+        if "+" + sym in x:
+            x = x.replace("+" + sym, "")
         elif x == sym:
-            x = '0'
+            x = "0"
     # handle special ENSDF abbreviations, e.g.,
     # http://www.iaea.org/inis/collection/NCLCollectionStore/_Public/14/785/14785563.pdf
     # "One of the following expressions:
     #   LT, GT, LE, GE, AP, CA, SY
     # for less than, greater than, less than or equal to greater
     # than or equal to. approximately equal to, calculated, and
     # from systematics, respectively."
-    for sym in ['*', '<', '>', '=', '~', '?', '@', '&', 'P', 'N']:
+    for sym in ["*", "<", ">", "=", "~", "?", "@", "&", "P", "N"]:
         while sym in x:
-            x = x.replace(sym, '')
+            x = x.replace(sym, "")
     # correct specific typos in the database
-    if 'E-11 0' in x:
-        x = x.replace('E-11 0', 'E-11')
-    if 'E-12 0' in x:
-        x = x.replace('E-12 0', 'E-12')
-    if '0.0000 1' in x:
-        x = x.replace('0.0000 1', '0.0000')
-    if '2 .8E-7' in x:
-        x = x.replace('2 .8E-7', '2.8E-7')
-    if '8 .0E-E5' in x:
-        x = x.replace('8 .0E-E5', '8.0E-5')
+    if "E-11 0" in x:
+        x = x.replace("E-11 0", "E-11")
+    if "E-12 0" in x:
+        x = x.replace("E-12 0", "E-12")
+    if "0.0000 1" in x:
+        x = x.replace("0.0000 1", "0.0000")
+    if "2 .8E-7" in x:
+        x = x.replace("2 .8E-7", "2.8E-7")
+    if "8 .0E-E5" in x:
+        x = x.replace("8 .0E-E5", "8.0E-5")
     # handle blank or missing data
-    if x == '' or x == ' ':
+    if x == "" or x == " ":
         return None
-    if '****' in dx:
-        dx = ''
-    elif dx in ['LT', 'GT', 'LE', 'GE', 'AP', 'CA', 'SY']:
-        dx = ''
+    if "****" in dx:
+        dx = ""
+    elif dx in ["LT", "GT", "LE", "GE", "AP", "CA", "SY"]:
+        dx = ""
     try:
         x2 = float(x)
     except ValueError:
-        raise NNDCRequestError(
-            'Value cannot be parsed as float: "{}"'.format(x))
-    if dx == '':
+        raise NNDCRequestError('Value cannot be parsed as float: "{}"'.format(x))
```

Comment:
Is it better to leave strings like these with single quotes if they enclose double quotes, or to make them doubles and escape the inner double quotes?

Reply:
```suggestion
        raise NNDCRequestError(f'Value cannot be parsed as float: "{x}"')
```
Need to move to the future :smile: No strong feelings about quotes. Think this is fine.

Reply:
Cleaned. b13c0aa52cd68cf402792f049bae90ab5096bf48 I'm good with the current formatting but can change. I think that is outside of this PR eh?

Reply:
Agreed, we can change it later if it bothers us.